packages/react-front-kit/src/3-custom/Components/Table/Table.tsx

```diff
@@ -0,0 +1,649 @@
+          </p>
+        </div>
+        <div style={{ marginTop: '32px' }}>
+          <Button
+            classNames={{
+              root: classes.buttonDontRemoveRoot,
+            }}
+            onClick={close}
+            style={{ marginRight: '10px' }}
+          >
+            Annuler
+          </Button>
+          <Button
+            classNames={{
+              root: classes.buttonRemoveRoot,
+            }}
+            onClick={() =>
+              sendCurrentElementValueWithAction(
+                currentElement,
+                'ADD_TO_FAVORITES',
+              )
+            }
+          >
+            Ajouter aux favoris
+          </Button>
+        </div>
+      </>,
+    );
+    open();
+  };
+  const shareHandle = (currentElement: IDocument): void => {
+    setModalContent(
+      <>
+        <div className="modal__header" style={{ marginLeft: '12px' }}>
+          <h2>Partager ?</h2>
+          <p>Êtes-vous certain de vouloir partager cette élément ?</p>
+        </div>
+        <div style={{ marginTop: '32px' }}>
+          <Button
+            classNames={{
+              root: classes.buttonDontRemoveRoot,
+            }}
+            onClick={close}
+            style={{ marginRight: '10px' }}
+          >
+            Ne pas partager
+          </Button>
+          <Button
+            classNames={{
+              root: classes.buttonRemoveRoot,
+            }}
+            onClick={() =>
+              sendCurrentElementValueWithAction(currentElement, 'SHARE')
+            }
+          >
+            Partager
+          </Button>
+        </div>
+      </>,
+    );
+    open();
+  };
+  const arboHandle = (currentElement: IDocument): void => {
+    // eslint-disable-next-line no-console
+    console.log(currentElement, 'ARBO_CHANGE_LOCATION');
+  };
+  const removeHandle = (currentElement: IDocument): void => {
+    setModalContent(
+      <>
+        <div className="modal__header" style={{ marginLeft: '12px' }}>
+          <h2>Supprimer ?</h2>
+          <p>Êtes-vous certain de vouloir supprimer cet élément ?</p>
+        </div>
+        <div style={{ marginTop: '32px' }}>
+          <Button
+            classNames={{
+              root: classes.buttonDontRemoveRoot,
+            }}
+            onClick={close}
+            style={{ marginRight: '10px' }}
+          >
+            Ne pas supprimer
+          </Button>
+          <Button
+            classNames={{
+              root: classes.buttonRemoveRoot,
+            }}
+            color="red"
+            onClick={() =>
+              sendCurrentElementValueWithAction(currentElement, 'REMOVE')
+            }
+          >
+            Supprimer
+          </Button>
+        </div>
+      </>,
+    );
+    open();
+  };
+
+  const multiRemoveHandle = (values: IDocument[]): void => {
+    setModalContent(
+      <>
+        <div className="modal__header" style={{ marginLeft: '12px' }}>
+          <h2>Supprimer ?</h2>
```

Comment:
Il faut éviter le Français en dur comme ça.
Je sais que pour l'instant c'était le moyen simple pour permettre de customiser la table et vérifier qu'on pouvait bien avoir ce que l'on veut.
Mais à terme en tout cas il faut que cela puisse être changé si jamais le site de l'utilisateur est en anglais par exemple.
Cependant cela fait beaucoup de mots Français un peu partout.
Il faut je pense voir comment on peut gérer ce système d'actions au global (car on doit pouvoir aussi utiliser la table dans d'autres contextes avec d'autres actions).

Reply:
Je ferme le commentaire mais c'est noté